src/index.ts

```diff
@@ -130,6 +130,8 @@ export const callToString = (calls: any[]): string =>
             const _currentValue =
                 typeof currentValue === 'string'
                     ? currentValue
+                    : currentValue.message !== undefined
```

Comment:
for readability reasons this statement could be shorten like this:
`: currentValue.message || JSON.stringify(currentValue);`
what is your opinion on this?

Reply:
yep, i fixed this